### DISCUSSION TITLE
挂载阿里云盘，打开以后是空白目录。

### DISCUSSION THREAD

**Discussion by Xia0WuYa** [AlistGo/alist#6935] [Q&A] [answered]
alist版本：v3.36.0
![版本](https://github.com/user-attachments/assets/d3c0733f-a388-4d06-b51c-e74195548fd8)

已经按照配置文档挂接了阿里云盘
![alist阿里网盘设置](https://github.com/user-attachments/assets/0117a7d8-dfea-4cb8-9cbb-8451eb3c8c64)

在alist中打开阿里云盘显示空白
![alist阿里盘空白](https://github.com/user-attachments/assets/afc0bd29-a2b3-484d-a9f6-1265461a85fc)
但其实阿里云盘里是有文件的
![阿里网盘内容](https://github.com/user-attachments/assets/24314fb4-ed31-4462-9bd1-36f58af8f559)

我挂载的123网盘是正常的
![123盘内容](https://github.com/user-attachments/assets/da472aeb-db4c-4d0c-b654-23eb701ea018)

有人知道是什么情况吗？ 谢谢！

**Comment by ririye** [comment]
由于原api https://api.nn.ci被sni阻断，导致国内无法连接的情况，故提供两个新的api供使用：

https://api-cf.nn.ci (套了一层cloudflare)
https://api.xhofe.top (辽宁多线， 由亚洲云赞助)
需要将aliyundrive_open后台配置中的Oauth令牌链接更换为

https://api-cf.nn.ci/alist/ali_open/token
https://api.xhofe.top/alist/ali_open/token

你的auth令牌链接没改

**Reply by Xia0WuYa** [reply]
@ririye 谢谢解答~  修改以后还是不行。
我一会新注册一个阿里账号试试~ 看是不是因为我的xiaoya挂的这个阿里云盘的原因。
稍后我再反馈~

**Reply by ririye** [reply, marked as answer]
云盘类型那里的默认改一下，改成资源库或备份盘，别用默认试试

**Reply by Xia0WuYa** [reply]
@ririye  更改云盘类型后成功~  3Q!  就是这个原因。

**Reply by HopeCollector** [reply]
> 云盘类型那里的默认改一下，改成资源库或备份盘，别用默认试试

你是我的神🙏
